src/lib/memory/MMU.cc

```diff
@@ -0,0 +1,36 @@
+#include "simeng/memory/MMU.hh"
+
+#include "simeng/OS/Constants.hh"
+namespace simeng {
+namespace memory {
+
+MMU::MMU(std::shared_ptr<Mem> memory, VAddrTranslator fn, uint64_t tid) {
+  memory_ = memory;
+  translate_ = fn;
+  tid_ = tid;
```

Comment:
Defined using initializer list, but what would be the benifit of defining the Constructor in the headers, won't that lead to stylistic inconsistency especially because the constructor isnt empty and that the MMU class doesn't define any other methods/functions in the headers, only declares them.

Reply:
As previous, agreed. Leave it where it is